crates/apollo-compiler/test_data/diagnostics/0056_variables_are_input_types.graphql

```diff
@@ -20,7 +20,7 @@ query takesCatOrDog($catOrDog: CatOrDog) {
   }
 }
 
-query takesCatOrDog($catOrDog: Dragon) {
+query takesCatOrDog2($catOrDog: Dragon) {
```

Comment:
@lrlna only noticed this because the error order changed in this PR … did this intentionally have a duplicate operation name?

Reply:
no, not intentional! human error